jina/orchestrate/deployments/__init__.py

```diff
@@ -284,6 +285,17 @@ def is_sandbox(self) -> bool:
         is_sandbox = uses.startswith('jinahub+sandbox://')
         return is_sandbox
 
+    @property
+    def _is_docker(self) -> bool:
+        """
+        Check if this deployment is to be run in docker.
+
+        :return: True if this deployment is to be run in docker
+        """
+        uses = getattr(self.args, 'uses', '')
+        is_docker = uses.startswith('jinahub+docker://') or uses.startswith('docker://')
```

Comment:
What about sandbox executors? I don't actually know, but I imagine they also run inside docker? Also more generally, can we know if external executors are dockerized? Or is it irrelevant in that case?